tests/runtimes/ios/ios_runtime_tests.py

```diff
@@ -290,3 +291,25 @@ def test_389_add_swift_files_to_xcode_project(self):
         result = Wait.until(lambda: all(string in File.read(log.log_file) for string in strings), timeout=300,
                             period=5)
         assert result, 'It seems that there\'s a problem with using swift files that are added in App_Resources'
+
+    def test_390_check_correct_name_of_internal_class_is_returned(self):
+        """
+        Test that NSStringFromClass function returns correct name of iOS internal class
+        https://github.com/NativeScript/ios-runtime/issues/1120
+        """
+
+        File.copy(os.path.join(TEST_RUN_HOME, 'assets', 'runtime', 'ios', 'files', 'ios-runtime-1120',
+                               'main-page.js'),
+                  os.path.join(APP_PATH, 'app', 'main-page.js'))
+
+        Tns.platform_remove(app_name=APP_NAME, platform=Platform.IOS)
```

Comment:
Why in all tests we do platform remove platform add?
Do we really need it before all tests?
If we do can we move it in hooks?

Can we think of separate tests that don't need platform rebuild in separate class where it happens only once?

Reply:
In the moment all ios tests are re creating the test project. After the release we are planing to implement logic for reverting changes made in the test which will remove the need of re creating the project and adding platforms and stuff like that. We have put it in our planing session.